src/storage/plugins/UdisksStorage.cxx

```diff
@@ -168,7 +168,7 @@ UdisksStorage::OnListReply(ODBus::Message reply) noexcept
 			if (!o.IsId(id))
 				return;
 
-			dbus_path = std::move(o.path);
+			dbus_path = o.path;
```

Comment:
This change is bad. You have let clang-tidy do its job without understanding the real problem.

Reply:
warning: std::move of the const expression has no effect; remove std::move() [hicpp-move-const-arg]

Reply:
The warning is correct, but not the solution. That's what I mean: you use clang-tidy blindly, without understanding why the problem is there, and let clang-tidy choose a solution which isn't good here.